### ISSUE TITLE
dofmricni loads not unarchived inplane anatomies

### ISSUE THREAD

**Issue by steevelaquitaine** [justingardner/gru#5]
dofmricni loads Inplane anatomies that are not unarchived (format .nii.gz). Clicking the files then produces .nii.gz.cpgz files. We cannot load the files as base anatomy.

**Comment by dbirman** [comment]
Until this is fixed you can re-download the .nii.gz files from NIMS, those files un-archive correctly into .nii files.

**Comment by steevelaquitaine** [comment]
Yes don't worry I did

**Comment by justingardner** [comment]
You should be able to load compressed nifti files (*.nii.gz) by doing:

[d h] = mlrImageLoad(‘filename.nii.gz’);

Then save it back uncompressed:

mlrImageSave(‘filename.nii’,d,h);

Also, I just fixed mrLoadRet so that it recognizes that you can load compressed nifti files, so you should be able to load from File/Base Anatomy/Load.

When you say you “clicked” on the files and got “.cpgz” files - I assume that means you clicked with the mouse from the finder? I wouldn’t do that ;)…

-j.

**Comment by steevelaquitaine** [comment]
Yes I clicked with the mouse. Ok.
Thanks !

**Comment by dbirman** [comment]
This fix doesn't work for me, I think that the files may be getting corrupted somewhere along the pipeline:

```
gunzip: anat01_10259_2_1_T1w_12mm_ax.nii.gz: not in gzip format
(cbiReadNiftiHeader) Incorrect header size (should be 348 bytes) for file anat01_10259_2_1_T1w_12mm_ax.nii
(mlrImageHeaderLoad) Could not load header for anat01_10259_2_1_T1w_12mm_ax.nii
```

**Comment by dbirman** [comment]
Re-downloading the file and clicking in the finder works fine by the way, so something is definitely going wrong somewhere in dofmricni

**Comment by dbirman** [comment]
Hmm, to add to the strangeness of this error it only occurs in some scans, some of my scans are unaffected... (i.e. the anat.nii file is already there unpacked)

**Comment by justingardner** [comment]
What scan is an example of the problem?

**Comment by dbirman** [comment]
s0300, scan 20150729, exam # 10255

**Comment by justingardner** [comment]
I ran dofmricni on exam 10255 and the anat file came out ok. wtf?

**Comment by dbirman** [comment]
Yeah that confirms my suspicion that the bug is fairly random. I get it in like 1 of 3 runs in all my data. Doesn't seem to be any consistency to what causes it.
